### ISSUE TITLE
API2 -  API - As an applicant, I would like to have "Name of Worker" within the add employee slider to be captured as First Name, Last Name to prevent data quality issues

### ISSUE THREAD

**Issue by binwang89** [18F/dol-whd-14c#714]
TODO - to be discussed.
Once API is connected to UI correctly, drop the “Name” column from DB and API. Blocked by the UI story

**Comment by binwang89** [comment]
See story #651 for reference

**Comment by PrabhakarThummalaDOL** [comment]
Closed. 
Duplicate of #767.
